.github/workflows/generate.yaml

```diff
@@ -0,0 +1,25 @@
+name: Generate
```

Comment:
Will this (and Validate) be run even if Compile fails? Can we make it wait until Compile succeeds before this is run?

(This is why I prefer them being separate steps in one action - while the PR description says it "Also makes the order of running these tools clear" it feels to me like it's doing the opposite.)

Reply:
I can merge them in one large action, although I would rather see 1/2 checks pass or 2/2 checks pass than 0/1 checks pass in the UI. Yes, the 2nd workflow (generate) cannot pass if the 1st workflow fails (compile).

The order is clearer as we have 4 tools in https://github.com/googleapis/google-cloudevents/tree/master/tools and we do not describe the specific order required to run them to correctly generate results. This CI does that, as you can just look at the CI and run the commands rather than looking at each tool.

Reply:
I think it's subjective, and I guess you'll be working in this repo more than I will be. I think the order is clearest if there's just *one* action that takes all the steps, but it's your call.